packages/terra-toggler/src/Toggler.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import AnimateHeight from 'react-animate-height';
+import 'terra-base/lib/baseStyles';
+import './Toggler.scss';
+
+const propTypes = {
+  /**
+   * Content in the body of the toggler component that will be expanded or collapsed
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * When set, will animate the toggler component as it is expanded or collapsed
+   */
+  isAnimated: PropTypes.bool,
+  /**
+   * Used to expand or collapse toggler content
+   */
+  isOpen: PropTypes.bool,
+};
+
+const defaultProps = {
+  children: null,
+  isAnimated: false,
+  isOpen: false,
+};
+
+const Toggler = ({ isAnimated, isOpen, children, ...customProps }) => {
+  const attributes = Object.assign({}, customProps);
+  const TogglerClassNames = classNames([
+    'terra-Toggler',
+    attributes.className,
+  ]);
+  const height = isOpen ? 'auto' : '0';
+  let body;
+
+  if (isAnimated) {
+    body = (
+      <AnimateHeight
+        duration={250}
+        height={height}
+      >
+        {children}
+      </AnimateHeight>
+    );
+  } else {
+    body = (
+      isOpen && children
+    );
+  }
+
+  return (
+    <div {...attributes} className={TogglerClassNames} aria-hidden={!isOpen}>
+      {body}
```

Comment:
Yeah, children needs to stay for the animation to have something to animate. The unanimated one only adds the children to the DOM if the [toggler is open](https://github.com/cerner/terra-core/blob/199038e9d93dc96e64c428906beb0fcb63d97336/packages/terra-toggler/src/Toggler.jsx#L49). If its closed, no children content exist in the DOM.

I've got some code to disable shifting focus to focusable elements within the toggler when its animated and the toggler is closed I'm getting ready to push up.